torch_np/_funcs_impl.py

```diff
@@ -913,6 +913,28 @@ def take_along_axis(arr: ArrayLike, indices: ArrayLike, axis):
     return torch.take_along_dim(arr, indices, axis)
 
 
+def put(a: ArrayLike, ind: Sequence[ArrayLike], v: ArrayLike, mode="raise"):
+    if mode != "raise":
+        raise NotImplementedError(f"{mode=}")
+
+    index = torch.concat(ind)
```

Comment:
According to NumPy's docs this is an array-like, so why using a sequence?

Reply:
Ah good shout, had thought of a few edge cases but didn't realise how powerful the `ArrayLike` normalisation is! So now using `ArrayLike` which clears up `put()` nicely and keeps `test_put` green :tada: